generator/internal/codegen_utils_test.cc

```diff
@@ -13,13 +13,19 @@
 // limitations under the License.
 
 #include "generator/internal/codegen_utils.h"
+#include <google/protobuf/descriptor.pb.h>
 #include <gmock/gmock.h>
 
 namespace google {
 namespace cloud {
 namespace generator_internal {
 namespace {
 
+using google::protobuf::DescriptorPool;
```

Comment:
Style: In using declarations, prefer to use fully qualify, e.g., `using ::foo::Bar;`